code/modules/mob/living/carbon/human/life.dm

```diff
@@ -363,8 +364,9 @@
 		bodytemperature += 11
 	else
 		bodytemperature += (BODYTEMP_HEATING_MAX + (fire_stacks * 12))
-		if(mind?.vampire && !mind.vampire.get_ability(/datum/vampire_passive/full) && stat != DEAD)
-			mind.vampire.bloodusable = max(mind.vampire.bloodusable - 5, 0)
+		var/datum/antagonist/vampire/V = mind?.has_antag_datum(/datum/antagonist/vampire)
+		if(V && V.get_ability(/datum/vampire_passive/full) && stat != DEAD) // V?. doesn't work here because `has_antag_datum` return FALSE if there is no datum and ?. doesn't like that
```

Comment:
```suggestion
		if(V && !V.get_ability(/datum/vampire_passive/full) && stat != DEAD) // V?. doesn't work here because `has_antag_datum` return FALSE if there is no datum and ?. doesn't like that
```